commcare-cloud-bootstrap/commcare_cloud_bootstrap.py

```diff
@@ -215,6 +215,10 @@ def ask_aws_for_instances(env_name, aws_config, count):
             subprocess.call(['commcare-cloud-bootstrap', 'terminate',  env_name])
             print("Deleting file: {}".format(cache_file))
             os.remove(cache_file)
+        env_dir = os.path.join(os.path.dirname(os.path.realpath(__file__)), '..', 'environments', env_name)
```

Comment:
would it be possible to construct this path without referencing `__file__`? otherwise if this code ever gets moved to a different place this would stop working and the person moving it might not realize it

Reply:
@pr33thi you can use `env.paths.paths.get_env_file_path('')`:

```
>>> from commcare_cloud.environment.main import get_environment
>>> get_environment('production').paths.get_env_file_path('')
'./production/'
```

Reply:
https://github.com/dimagi/commcare-cloud/pull/1996/commits/240586a654b15d4a572a3da366870d704561190b